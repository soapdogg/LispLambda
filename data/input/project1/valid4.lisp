(DEFUN TRIPLE (X)(3 X Y))