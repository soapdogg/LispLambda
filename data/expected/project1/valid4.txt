LITERAL ATOMS: 5, DEFUN, TRIPLE, X, X, Y
NUMERIC ATOMS: 1,3
OPEN PARENTHESES: 3
CLOSING PARENTHESES: 3